src/styles/index.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  margin: 0;
+  font-family: 
+  /* -apple-system, BlinkMacSystemFont,  */
+    'Work Sans', 'Roboto Slab', 'Segoe UI', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  font-weight: medium;
+  background-image: linear-gradient(
+    45deg,
+    rgba(0, 162, 194, 0.6) 0%,
+    rgba(0, 171, 194, 0.6) 12%,
+    rgba(0, 188, 194, 0.6) 25%,
+    rgba(0, 199, 179, 0.6) 40%,
+    rgba(0, 209, 160, 0.6) 67%,
+    rgba(46, 220, 139, 0.6) 77%,
+    rgba(109, 227, 113, 0.6) 83%,
+    rgba(158, 233, 83, 0.6) 89%,
+    rgba(204, 235, 51, 0.6) 95%,
+    rgba(255, 234, 0, 0.6) 100%
+), url('../../public/images/rainy-rainbow.webp');
```

Comment:
Its not a usual format for an image here... would recommend swapping this for a jpg